cmd/metrics-realtime.go

```diff
@@ -41,6 +43,22 @@ func collectLocalMetrics(types madmin.MetricType, opts collectMetricsOpts) (m ma
 		return
 	}
 
+	byHostName := globalMinioAddr
```

Comment:
```suggestion
	hostName := globalMinioAddr
```
otherwise changes look fine

Reply:
@harshavardhana Need a name here.

Reply:
byHostName is kind of an query variable name so in this context it's okay @shtripat